### PR TITLE
Disable TTS on embedded levels

### PR DESCRIPTION
**Pull request by epeach**
Only enable TTS button on levels that have markdown instructions and which are in a script that had been approved for TTS.

### CODE REVIEW

**Comment by epeach** [comment]
The markdown instructions are played by the current TTS set-up. So a level with just tts_instructions_text and no markdown instructions will currently display an icon for TTS but have no accompanying audio track. This change will make it so that such levels will not display a TTS icon ( as well as embedded levels)

**Comment by Hamms** [comment]
Note that this method is not used just by embedded levels, but also by levels which support a combination of markdown and regular instructions tracks.

If I'm reading it right, this change would disable TTS for levels like https://studio.code.org/s/coursea/stage/4/puzzle/2 that have only the regular instructions.

**Comment by epeach** [comment]
I see - I didn't know we supported levels like that. Thanks for pointing that out! I will edit this PR accordingly.

**Comment by epeach** [comment]
@Hamms PTAL. Thank you!

**Comment by epeach** [comment]
This change was requested to remove tts from levels for which there is no matching audio recording on embedded levels. I believe it is going to be a temporary change until we can get audio recordings for embedded levels.

Your proposed method is safer in regards to the original behavior of TTS. Updating now.

**Comment by epeach** [comment]
@Hamms PTAL.

File: apps/src/templates/instructions/TopInstructionsCSP.jsx [hunk 1]
```diff
@@ -223,7 +223,7 @@ var TopInstructions = React.createClass({
       <div style={mainStyle} className="editor-column">
         <PaneHeader hasFocus={false}>
           <div style={styles.paneHeaderOverride}>
-            {!this.state.helpTabSelected &&
```
Comment (epeach) [inline apps/src/templates/instructions/TopInstructionsCSP.jsx:226]: (Note: CSF already protects itself against empty strings (in ChatBubble). )

**Pull request merged**